code/modules/cargo/packs.dm

```diff
@@ -128,20 +128,48 @@ var/list/all_supply_groups = list("Operations","Security","Hospitality","Enginee
 	group = "Security"
 	hidden = TRUE
 
-/datum/supply_pack/weapons
-	name = "Weapons crate"
-	contains = list(/obj/item/weapon/melee/baton,
-					/obj/item/weapon/melee/baton,
-					/obj/item/weapon/gun/energy/laser,
-					/obj/item/weapon/gun/energy/laser,
-					/obj/item/weapon/gun/energy/taser,
-					/obj/item/weapon/gun/energy/taser,
-					/obj/item/weapon/storage/box/flashbangs,
-					/obj/item/weapon/storage/box/flashbangs)
+/datum/supply_pack/fsenergy
+	name = "FS Energy Weapons"
+	contains = list(/obj/item/weapon/gun/energy/cassad,
+				/obj/item/weapon/gun/energy/gun,
+				/obj/item/weapon/gun/energy/gun,
+				/obj/item/weapon/gun/energy/gun/martin,
+				/obj/item/weapon/gun/energy/gun/martin)
+	cost = 3500
+	containertype = /obj/structure/closet/crate/secure/weapon
+	crate_name = "FS Energy Weapons"
+	group = "Security"
+	
+/datum/supply_pack/fssmall
+	name = "FS Handgun Pack"
+	contains = list(/obj/item/weapon/gun/projectile/colt,
+			/obj/item/weapon/gun/projectile/IH_sidearm,
+			/obj/item/weapon/gun/projectile/clarissa,
+			/obj/item/weapon/gun/projectile/olivaw)
 	cost = 3000
 	containertype = /obj/structure/closet/crate/secure/weapon
-	crate_name = "Weapons crate"
-	access = access_brig
+	crate_name = "FS Handgun Pack"
+	group = "Security"
+	
+/datum/supply_pack/fsassault
+	name = "FS Assault Pack"
+	contains = list(/obj/item/weapon/gun/projectile/automatic/ak47/fs,
+			/obj/item/weapon/gun/projectile/automatic/wt550,
+			/obj/item/weapon/gun/projectile/automatic/wt550)
```

Comment:
I'm not sure if these smgs are using proper updated sprite for gun and ammo, maybe it's better to use some other option here

Reply:
My bad, I confused the Idaho (the top mounted 9mm one, I believe) and the Wt550.